src/main/java/com/hms_networks/sc/cumulocity/api/CConnectorProvisionMqttMgr.java

```diff
@@ -0,0 +1,195 @@
+package com.hms_networks.sc.cumulocity.api;
+
+import com.ewon.ewonitf.DefaultEventHandler;
+import com.ewon.ewonitf.MqttMessage;
+import com.hms_networks.americas.sc.extensions.json.JSONException;
+import com.hms_networks.americas.sc.extensions.logging.Logger;
+import com.hms_networks.americas.sc.extensions.mqtt.MqttManager;
+import com.hms_networks.americas.sc.extensions.string.StringUtils;
+import com.hms_networks.sc.cumulocity.CConnectorMain;
+import java.util.List;
+
+/**
+ * The MQTT management class for the Cumulocity Connector.
+ *
+ * @author HMS Networks, MU Americas Solution Center
+ * @since 1.0.0
+ */
+public class CConnectorProvisionMqttMgr extends MqttManager {
+
+  /** QOS Level for MQTT connections. Azure IoT Hub requires QOS Level 1. */
+  public static final int MQTT_QOS_LEVEL = 1;
+
+  /** TLS version for MQTT connections. Azure IoT Hub connections use TLS v1.2. */
+  public static final String MQTT_TLS_VERSION = "tlsv1.2";
+
+  /** The MQTT topic for publishing messages to Cumulocity via MQTT. */
+  private static final String CUMULOCITY_MQTT_TOPIC_UCR = "s/ucr";
+
+  /** The MQTT topic for receiving messages from Cumulocity via MQTT. */
+  private static final String CUMULOCITY_MQTT_TOPIC_DCR = "s/dcr";
+
+  /** The message ID used for MQTT device provisioning messages with credentials. */
+  private static final String CUMULOCITY_MQTT_PROVISION_CREDENTIALS_MESSAGE_ID = "70";
+
+  /** The maximum number of seconds between retries for MQTT connections. */
+  public static final long MAX_SECONDS_BETWEEN_RETRIES = 180;
+
+  /**
+   * The time (in milliseconds) to wait between each MQTT loop. Changing this will modify the
+   * interval at which payloads are checked for sending.
+   */
+  public static final long MQTT_LOOP_WAIT_MILLIS = 5000;
```

Comment:
is this too lax. Should be 2500?

Reply:
It could be shortened, but I left this connector at a slightly increased time because there is no need for a payload management system here, which removes a lot of delays that might've been introduced by other components under other scenarios.